src/components/FilterCollapse.js

```diff
@@ -131,6 +131,10 @@ class FilterCollapse extends Component {
     const tokens = [];
     for (const name of allFields) {
       if (this.state[name]) {
+        if (name == "status")
+        {
+          this.state[name] = "\"" + this.state[name] + "\"";
+        }
```

Comment:
So this pattern doesn't work in React - you need to use `this.setState` to change component state in React, and anyway you don't want to change the component state here (you don't want to put the quotes in the Advanced modal). But since this change is included in #542, I think you can just discard this change and the other one in this file.

Reply:
I think you still need to remove this change